src/Library/demos/Video/main.blp

```diff
@@ -0,0 +1,30 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage page {
+  title: "Video";
+  description: _("A Widget that displays video with media controls");
+
+  Box {
+    orientation: vertical;
+    halign: center;
+
+    Video video {
+      width-request: 640;
+      height-request: 360;
+    }
+
+    Button file_button {
+      halign: center;
+      label: _("Load Video File");
+      margin-top: 12;
+    }
```

Comment:
let's ship a video and load it directly instead

you can use this one for now, I have asked Jakub if we can use it

https://github.com/sonnyp/Workbench/assets/19673/2f35ff89-64b7-4939-9ceb-ee93b29159ad

Reply:
Check the overlay demo to see how to load a file

Reply:
Jakub agreed to us using this in the library under CC0 \o/

Can you add them to `artists` credits in `about.js` ? `Jakub Steiner https://jimmac.eu`

Reply:
EDIT: nevermind I added Jakub in https://github.com/sonnyp/Workbench/pull/333/files